nautobot/dcim/signals.py

```diff
@@ -18,7 +18,7 @@
 )
 
 
-def create_cablepath(node):
+def create_cablepath(node, rebuild=True):
```

Comment:
Since this behavior can be confusing, can you please add to the docstring here explaining what the `rebuild` argument does?